components/count-badge/count-badge-mixin.js

```diff
@@ -198,7 +198,8 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(S
 		if (!numberStyles || numberStyles.visibility !== 'hidden') {
 			numberStyles = {
 				...numberStyles,
-				visibility: hideNumber ? 'hidden' : 'visible'
+				visibility: hideNumber ? 'hidden' : 'visible',
+				zIndex: 1
```

Comment:
Is this `z-index` and the `isolation` below to address the same issue, or something different?